unit_tests/test_events.py

```diff
@@ -20,32 +20,32 @@
 
 class SctEventsTests(unittest.TestCase):  # pylint: disable=too-many-public-methods
 
-    def get_event_logs(self):
-        log_file = os.path.join(self.temp_dir, 'events_log', 'events.log')
+    @classmethod
+    def get_event_logs(cls):
+        log_file = os.path.join(cls.temp_dir, 'events_log', 'events.log')
```

Comment:
we are in Python 3, use Path from pathlib
```
log_file = Path(d) /'events_log'/ 'events.log'
if  log_file.exists():
    with log_file.open(mode="r"):
        return file.read()
return ""
```

Reply:
Fixed